harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/geoPREST/Harvester.java

```diff
@@ -262,6 +265,33 @@ private RecordInfo getRecordInfo(Element record) {
 
     }
 
+    protected Date parseDateJdk11(String pubDate) throws ParseException {
```

Comment:
Since we can't compile the code for Java 8 anymore why do we have a new `parseDateJdk11` instead of replacing the code of the old `parseDate`?